QUICK_TEST_REFERENCE.md

```
@@ -0,0 +1,58 @@
+# Quick Test Reference Guide
+
+## Running Tests
+
+### Basic Commands
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)